scripts/run_kickstart_tests.sh

```diff
@@ -350,11 +355,8 @@ if [[ "$TEST_REMOTES" != "" ]]; then
             scp -r kstest@${remote}:/var/tmp/kstest-\* /var/tmp/
         fi
 
-        ssh kstest@${remote} sudo rm -rf kickstart-tests /var/tmp/kstest-\*
+        ssh kstest@${remote} sudo rm -rf /var/tmp/kstest-\*
     done
-
-    # (3a) And then also remove the copy of the image we made earlier.
-    rm ${_IMAGE}
```

Comment:
Why are you removing this?

Reply:
We are running rsync with  --delete on the kickstart-tests folder, so we don't need to explicitly delete the ISO file anymore - rsync will do that for us when it takes care of any leftovers in the folder on the next run.

Reply:
I'm not that sure, this is local system call `rm` but those rsync calls are for remote machines. Is that really working correctly?

Reply:
This is to support the `TEST_REMOTES_ONLY` != yes usecase, where also the machine that started the run is running some of the tests locally. This way you don't need to have a dedicated management machine/VM and all machines can run tests.

The side effect of that is that some things need to be done twice, once for all the remotes and once for the local machine, if `TEST_REMOTES_ONLY` != yes. The resulting environment then needs to be substantially similar so that run_one_tests.py can run the same on the remotes and one the local machine.

As for the removed rm call - we now copy the image file ti install_images folder instead of directly to the kickstart-tests folder:
```
-    cp ${IMAGE} ${_IMAGE}
+    mkdir -p ../install_images
+    cp ${IMAGE} ../install_images/
```
Therefore the rm call on `${_IMAGE}` is no longer relevant as `${_IMAGE}` will not be there).